src/components/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+
+import ComponentRegister, { LinkComponent } from "../ComponentRegister";
+
+import "lbh-frontend/lbh/core/_links.scss";
+
+/**
+ * Generates a Link component that's a wrapper for the Link attribute of
+ * ComponentRegister
+ * @param {string} [className] - A string of class names
+ * @param {string} href - A url to direct the user if the link is clicked
```

Comment:
```suggestion
 * @param {string} href - The url to direct the user to if the link is clicked
```

Prefer "the" over "a" for required parameters.